src/Plug.vala

```diff
@@ -21,6 +21,11 @@ public class DateTime.Plug : Switchboard.Plug {
     private MainView main_view;
 
     public Plug () {
+        GLib.Intl.setlocale (LocaleCategory.ALL, "");
+        GLib.Intl.bindtextdomain (GETTEXT_PACKAGE, LOCALEDIR);
+        GLib.Intl.bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
+        GLib.Intl.textdomain (GETTEXT_PACKAGE);
```

Comment:
```suggestion
        GLib.Intl.bindtextdomain (GETTEXT_PACKAGE, LOCALEDIR);
        GLib.Intl.bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
```
For Switchboard plugs and Wingpanel indicators, we should only use these two functions, the rest is already set by wingpanel and switchboard themselves